src/pages/ReimbursementAccount/AddressForm.js

```diff
@@ -139,7 +139,7 @@ const AddressForm = props => (
             onChangeText={value => props.onFieldChange({zipCode: value})}
             errorText={props.errors.zipCode ? props.translate('bankAccount.error.zipCode') : ''}
             maxLength={CONST.BANK_ACCOUNT.MAX_LENGTH.ZIP_CODE}
-            hint={props.translate('common.zipCodeExample')}
+            hint={props.translate('common.format', {zipSampleFormat: CONST.COUNTRY_ZIP_REGEX_DATA.US.samples})}
```

Comment:
On Home Adress page, there is no hint displayed by default.
It only appears when you use address search to select an address. 

<img width="399" alt="image" src="https://user-images.githubusercontent.com/29673073/230882362-432c19bc-dc02-4ce0-8109-c234bb4ac714.png">